concent_api/api-test.py

```diff
@@ -1,72 +1,152 @@
 #!/usr/bin/env python3
 
 import sys
-import json
 import datetime
 import random
+import http.client
+from base64                             import b64encode
+
+from golem_messages.message             import Message, MessageForceReportComputedTask, MessageRejectReportComputedTask, MessageCannotComputeTask, MessageTaskToCompute, MessageAckReportComputedTask
```

Comment:
We want to introduce some changes in imports. Generally two rules:
1. Try to keep alphabetical order in imports group.
2. When doing many imports from single module, split each one into separate line, like this:
```
from golem_messages.message import Message
from golem_messages.message import MessageForceReportComputedTask
from golem_messages.message import MessageRejectReportComputedTask
....
```